### ISSUE TITLE
Identify the AI Architecture and method that will be using in the AI service

### ISSUE THREAD

**Issue by rangsey-Rv** [SirOsborn/haniphei.ai#5]
conduct research on what AI technology that is suitable for the document scanning mechanism 

- [x] OCR to extract text
- [ ] LLM approach (fast to implement but high cost as it use per tokens analysis)
- [ ] Fine-tuning a pretrained model (legalBERT, generalBERT) or transformers with specific datasets (longer implementation and complicated but lower operational cost full control over data privacy)
- [x] Hybrid AI architecture approach start with LLM (need to define what LLM to use) then continuously fine-tuning a pretrained model for long term use. Can stop using LLM completely after accuracy of the mode is commercially satisfied
